.npmrc

```diff
@@ -4,11 +4,11 @@ link-workspace-packages=true
 save-workspace-protocol=false # This prevents the examples to have the `workspace:` prefix
 auto-install-peers=false
 
-# `github-slugger` is used by `vite-plugin-markdown-legacy`.
-# Temporarily hoist this until we remove the feature.
-public-hoist-pattern[]=github-slugger
```

Comment:
This is no longer needed as we had removed the `vite-plugin-markdown-legacy` plugin